test/tests/functional/pbs_Rrecord_resources_used.py

```diff
@@ -543,14 +543,14 @@ def test_Rrecord_job_rerun_forcefully(self):
 
```

Comment:
The test comment says the run_count should be 2....but the match searches for run_count=1?  Which is correct?  Please fix whichever is wrong.